internal/work/results/compute.go

```diff
@@ -354,7 +370,7 @@ func (c *Computation) EvaluateWorkPackage(
 // ComputePagedProofs P(s) → [E(J₆(s,i), L₆(s,i))₍l₎ | i ∈ ℕ₍⌈|s|/64⌉₎] (14.10 v0.5.4)
 func ComputePagedProofs(segments []work.Segment) ([]work.Segment, error) {
 	if len(segments) == 0 {
-		return nil, fmt.Errorf("no segments provided")
+		return nil, nil
```

Comment:
This shouldn't be an error?

Reply:
yes, because it is possible that after the execution we get no segments, this should not fail the process